minds/minds.py

```diff
@@ -216,7 +222,9 @@ def get(self, name: str) -> Mind:
         :param name: name of the mind
         :return: a mind object
         """
-
+        if not utils.validate_mind_name(name):
```

Comment:
let's remove it and check for `.drop`
let's keep check of name only for cases when this name is going to be stored in database:
- create
- update